ios/brave-ios/Sources/BraveVPN/Components/Region/BraveVPNRegionConfirmationView.swift

```diff
@@ -0,0 +1,129 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveShared
+import BraveUI
+import Shared
+import SwiftUI
+
+public struct BraveVPNRegionConfirmationView: View {
+  let regionCountry: String?
+  let regionCity: String?
+  let regionCountryFlag: Image?
+
+  public init(regionCountry: String?, regionCity: String?, regionCountryISOCode: String?) {
+    self.regionCountry = regionCountry
+    self.regionCity = regionCity
+    self.regionCountryFlag = regionCountryISOCode?.regionFlag ?? Image(braveSystemName: "leo.globe")
+  }
+
+  public var body: some View {
+    VStack(spacing: 24) {
+      Image("region-change-banner", bundle: .module)
+        .resizable()
+        .frame(width: 96, height: 96)
+      Text(Strings.VPN.vpnRegionChangedTitle)
+        .multilineTextAlignment(.center)
+        .font(.headline)
+        .foregroundStyle(Color(braveSystemName: .textPrimary))
+      HStack {
+        if let regionCountryFlag = regionCountryFlag {
+          regionCountryFlag
+            .frame(width: 32, height: 32)
+            .padding(4)
+            .overlay(
+              RoundedRectangle(cornerRadius: 12.0, style: .continuous)
+                .strokeBorder(Color(.lightGray), lineWidth: 1.0)
+            )
+        }
+        VStack(alignment: .leading) {
+          if let regionCountry = regionCountry {
+            Text(regionCountry)
+              .font(.headline)
+              .foregroundStyle(Color(braveSystemName: .textPrimary))
+          }
+          if let regionCity = regionCity {
+            Text(regionCity)
+              .font(.subheadline)
+              .foregroundStyle(Color(braveSystemName: .textSecondary))
+          }
+        }
+        Spacer()
+      }
+    }
+    .padding(48)
+    .background(Color(braveSystemName: .containerBackgroundMobile))
+    .frame(maxWidth: 350)
+  }
+}
+
+struct BraveVPNRegionConfirmationView_Previews: PreviewProvider {
+  static var previews: some View {
+    BraveVPNRegionConfirmationView(
+      regionCountry: "Canada",
+      regionCity: "Saskatchewan",
+      regionCountryISOCode: "CA"
```

Comment:
why is this hardcoded?

Reply:
do we need it in the production code?

Reply:
oh good catch it needs #if DEBUG on top of it

Reply:
Fix at https://github.com/brave/brave-core/pull/24198/commits/2bf4647300a20d601cf1d31485b4cf86586f030e